frontend/src/app/sort-input/sort-input.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-form-field class="sort-select" appearance="fill">
+    <mat-label>{{'SORT.TITLE' | translate}}</mat-label>
+    <mat-select [(ngModel)]="sortMethod" (selectionChange)="selectMethod(this.sortMethod)">
+      <mat-option *ngFor="let sortMethod of options" [value]="sortMethod.value">
```

Comment:
**issue:** `sortMethod` is shadowed here (used for different purposes on lines 3 and 4)